behind-the-veil-siteroot/imports/ui/components/map/CenteredMap.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * File Description: A simple map component with a center at given location
+ * File version: 1.0
+ * Contributors: Laura
+ */
+
+import React, { useEffect, useState, useRef } from 'react';
+import mapboxgl from 'mapbox-gl';
```

Comment:
Please run `npm i mapbox-gl` in your terminal to add mapbox as a dependency for our project. Once you run the command, you should see changes to your package.json and your package-lock.json files. Then, commit those changes.

Currently, others who pull from this branch and run `meteor npm install` and `meteor run` will see the following error b/c the package hasn't been added as a dependency
<img width="549" alt="Screenshot 2024-05-07 at 5 32 38 pm" src="https://github.com/Monash-FIT3170/Behind-The-Veil/assets/95511987/210274e5-5e80-4e90-b1af-9612cb644e4d">